### PR TITLE
feat: monitoring resources for experimentation, doc update

### LINKED ISSUE #713: feat: monitoring resources
the `core-landing-zone` packages are missing basic resources to ease enablement of monitoring

- proper IAM permissions to logging namespace
- workload identity resources to manually enable Config Sync monitoring
- metrics scope to centralize management of alerts

[documentation](https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/blob/main/docs/landing-zone-v2/README.md) should also be updated to make monitoring manual and optional

### PR DESCRIPTION
**Pull request by alaincormier-ssc**
closes #713

- add monitoring resources to experimentation
- update documentation

### CODE REVIEW

File: docs/landing-zone-v2/README.md [hunk 1]
```diff
@@ -615,6 +616,70 @@ nomos status --contexts gke_${PROJECT_ID}_northamerica-northeast1_krmapihost-${C
...
+    > **Note:** Since Config Sync version 1.15.0, depending on the frequency of production repository commits and variance in Kubernetes Kinds being deployed, the two new metric labels `commit` and `type` introduced in this release can cause an increase in Cloud Monitoring metric write throughput. This can result in an increase of monthly billing. The affected metrics are `apply_duration_seconds`, `api_duration_seconds`, `declared_resources`, `apply_operations_total`. To avoid an increase in billing or if these metrics are not needed, follow the instructions to [patch the otel collector deployment with ConfigMap](https://github.com/GoogleContainerTools/kpt-config-sync/blob/v1.15/docs/custom-metric-filter.md#patch-the-otel-collector-deployment-with-configmap) and remove the aforementioned metrics from `filter/cloudmonitoring` allowlist.
+
+    If monitoring is no longer required, the annotation can be removed with the command below:
+    ```bash
+    # remove the annotation
+    kubectl annotate serviceaccount \
+      --namespace config-management-monitoring \
+      default \
+      iam.gke.io/gcp-service-account-
```
Comment (fmichaelobrien) [inline docs/landing-zone-v2/README.md:654]: Looking up the kubectl call but I thnk the service account is truncated before and expected gcp-service-account=...
at gcp-service-account-

Comment (fmichaelobrien) [reply docs/landing-zone-v2/README.md:654]: never mind - I checked, to remove the SA place a - minus at the end
all good - adding this one to my queue

https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity

**Review by fmichaelobrien — APPROVED** [review]
/lgtm

**Review by cartyc — APPROVED** [review]

**Review by lucstjean-ssc — APPROVED** [review]
/lgtm

**Pull request merged**